src/Arcus.Templates.WebApi/Controllers/HealthController.cs

```diff
@@ -35,6 +38,10 @@ public HealthController(HealthCheckService healthCheckService)
         [HttpGet(Name = "Health_Get")]
         [ProducesResponseType(typeof(HealthReport), StatusCodes.Status200OK)]
         [ProducesResponseType(typeof(HealthReport), StatusCodes.Status503ServiceUnavailable)]
+#if (ExcludeCorrelation == false && ExcludeCorrelation == false)
+        [SwaggerResponseHeader(200, "RequestId", "string", "The header that will have an unique identifier to distinguish between requests/responses")]
+        [SwaggerResponseHeader(200, "X-Transaction-Id", "string", "The header that will have an identifier that relate requests/responses")]
```

Comment:
What about this, isn't this more clear?
```suggestion
        [SwaggerResponseHeader(200, "X-Transaction-Id", "string", "The header that has the transaction ID is used to correlate multiple operation calls.")]
```